benchmarks/pytorch-lightning/giving_callback.py

```diff
@@ -0,0 +1,95 @@
+""" PyTorch-Lightning callback that uses giving.give. Can be used on basically any model. """
```

Comment:
Ohhh... that's neat. I might pull this into milabench proper somewhere so that the instrumentation could be reused for other lightning code.